.github/workflows/memory-testing.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Memory Testing
+
+on:
+  pull_request:
+    paths:
+      - "crates/bitwarden-crypto/**"
+      - "crates/memory-testing/**"
+  push:
+    paths:
+      - "crates/bitwarden-crypto/**"
+      - "crates/memory-testing/**"
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  msrv:
+    name: Memory Testing for - ${{ matrix.settings.os }} - ${{ matrix.settings.target }}
+    runs-on: ${{ matrix.settings.os || 'ubuntu-latest' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        settings:
+          - os: ubuntu-22.04
+            target: x86_64-unknown-linux-gnu
```

Comment:
Do you think we'll expand this to other architectures? If not lets remove the matrix and just run it on ubuntu.

Reply:
At the moment this is linux only, but we could also run it on arm linux. That said none of the code we're handling deals with anything os or architecture specific so there might be no reason to.